scss/_reboot.scss

```diff
@@ -19,7 +19,7 @@
 // 6. Change the default tap highlight to be completely transparent in iOS.
 
 html {
-  box-sizing: border-box; // 1
+  box-sizing: inherit; // 1
```

Comment:
Ok I am on it. I don't think we need to update comments. As the comments are still correct.